### ISSUE TITLE
VariableforProperty shouldn't trigger when using a function or mixin

### ISSUE THREAD

**Issue by nothingrandom** [brigade/scss-lint#911]
It's quite common to use a palette function for colors.

```
// pallette
// @see https://twitter.com/thecodezombie/status/595977653296996354/photo/1
@function palette($palette, $tone: 'base') {
  @return map-get(map-get($palettes, $palette), $tone);
}
```

```
$palettes: (
	grey: (
		border: #E8E8E8,
		text: #565656
	)
);
```

```
div {
    border: 1px solid palette(grey, border);
    color: palette(grey, text);
}
```

These both currently trigger a linter error if you've got either ColorVariable or VariableforProperty enabled.

**Comment by sds** [comment]
Duplicate of #559. Happy to accept a pull request!

**Comment by sds** [comment]
My mistake. This is actually a duplicate of #821.
